tests/integration_rust/install_filter_tests.rs

```diff
@@ -0,0 +1,205 @@
+use pixi_core::{InstallFilter, UpdateLockFileOptions};
```

Comment:
This contains some integration tests, with an e2e test to check if all works as expected.